applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_simple_steady_scheme.h

```diff
@@ -376,14 +364,7 @@ class ResidualBasedSimpleSteadyScheme : public Scheme<TSparseSpace, TDenseSpace>
 
       for (unsigned int i = 0; i < Dimension; i++)
       {
-        double LumpedMass = 0.0;
-        for (unsigned int j = 0; j < VelocityBlockSize; j++)
-        {
-          LumpedMass += Mass(DofIndex,j);
-          Mass(DofIndex,j) = 0.0;
-        }
-        // the relaxation factor defines the local cfl number
-        Mass(DofIndex,DofIndex) = LumpedMass / (mVelocityRelaxationFactor * LocalDt);
+        Mass(DofIndex,DofIndex) /= (mVelocityRelaxationFactor * LocalDt);
```

Comment:
The compiler might optimize this better if we do `*= 1.0 / (...)`. Apparently, the compiler may not substitute multiplication with reciprocal by default since it can produce a slightly different result.